atlasdb-cassandra-integration-tests/build.gradle

```diff
@@ -52,6 +52,9 @@ dependencies {
 
     /* TODO(boyoruk): Upgrade all package to JUnit5. */
     testImplementation 'junit:junit'
+    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine', {
+        because 'allows JUnit 3 and JUnit 4 tests to run'
+    }
```

Comment:
They stopped working when we add 
`implementation 'org.junit.jupiter:junit-jupiter'` to `atlasdb-tests-shared` package.